Source/FluentSpreadsheets/Visitors/ComponentVisitor.cs

```diff
@@ -0,0 +1,134 @@
+namespace FluentSpreadsheets.Visitors;
+
+public class ComponentVisitor<THandler> : IComponentVisitor where THandler : IComponentVisitorHandler
```

Comment:
```suggestion
public class ComponentVisitor<THandler> : IComponentVisitor
    where THandler : IComponentVisitorHandler
```

Reply:
single generic parameter constraint looks dangling on next line, so it is better to keep it on the same line.

Reply:
Okay, but imho, its just hard to read in single line